### DISCUSSION TITLE
ModuleNotFoundError: No module named 'pyserini.search.lucene'

### DISCUSSION THREAD

**Discussion by xiny-gao** [castorini/pyserini#2043] [Q&A] [answered]
I tried to install pyserini by using "pip install pyserini", but when using following code:

from pyserini.search.lucene import LuceneSearcher

I received an error:
ModuleNotFoundError: No module named 'pyserini.search.lucene'

How to fix it?
python3.8 + jdk11 + pyserini==0.12.0
python3.10 + jdk21 + pyserini==0.12.0
The two environment where I encountered the same problems.

**Comment by lintool** [comment]
The latest version of pyserini is 0.43.0: https://pypi.org/project/pyserini/
Please try this version.

**Reply by xiny-gao** [reply, marked as answer]
Thanks! I just tried python=3.10+jdk21+pyserini==0.16.0 and the problem was solved!
